app/components/blogFilter.tsx

```diff
@@ -0,0 +1,83 @@
+'use client'; // Tento řádek je potřeba, aby byla komponenta považována za klientskou
+
+import React, { useState } from 'react';
```

Comment:
Also I would put this file to app/blog/components as it is only tied to blog pages